<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuelLevelClass</class>
 <widget class="QWidget" name="FuelLevelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>425</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>FuelLevel</string>
  </property>
  <widget class="QLabel" name="redLight">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/icons/fuelIconRed.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="yellowLight">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="icons.qrc">:/icons/fuelIconYellow.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="fuelPercent">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Fuel</string>
   </property>
   <property name="fuelInput" stdset="0">
    <string>100</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="fuelBar">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>51</width>
     <height>141</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
